completion_aggregator/api/v1/views.py

```diff
@@ -363,6 +359,9 @@ def get(self, request, course_key):
             # Use enrollments for the effective user
             enrollments = UserEnrollments(self.user).get_course_enrollments(course_key)
 
+        if 'user_ids' in request.query_params:
+            user_ids = (int(id) for id in request.query_params['user_ids'].split(','))
+            enrollments = enrollments.filter(user_id__in=user_ids)
```

Comment:
@jcdyer @xitij2000 I think this ends up looking like 1 query per user:

![selection_002](https://user-images.githubusercontent.com/10018065/48531519-632b1c00-e8be-11e8-82be-becbbfce4027.jpg)
![selection_001](https://user-images.githubusercontent.com/10018065/48531526-67573980-e8be-11e8-9600-ccea122b9eaa.jpg)

Reply:
@UmanShahzad Hmm. I think that's probably because just a little later we have ``user__in=[enrollment.user for enrollment in paginated]`` this could definitely be optimised.

Reply:
Fixed it with a `.select_related('user')` here - it's taking 9 queries only now, and taking only ~300ms-1second for a 200 user batch. (My devstack is extremely slow because of VBoxFS and whatever, so in production this is way faster, 100ms probably).